addons/misra.py

```diff
@@ -2581,8 +2581,6 @@ def misra_11_6(self, data):
         for token in data.tokenlist:
             if not isCast(token):
                 continue
-            if token.astOperand1.astOperand1:
-                continue
```

Comment:
If this was here for a specific reason I'm happy to address it. I didn't understand why it was here so I removed it.

Reply:
I don't know why it was here.